### PR TITLE
add an example using environment variables

### PR DESCRIPTION
**Pull request by gtfisher**
Signed-off-by: gtfisher <grant@chaosiq.io>

### CODE REVIEW

File: examples/basic-env-vars.yaml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    }
+---
+apiVersion: chaostoolkit.org/v1
+kind: ChaosToolkitExperiment
+metadata:
+  name: my-chaos-exp
+  namespace: chaostoolkit-crd
+spec:
+  namespace: chaostoolkit-run
+  pod:
```
Comment (dmartin35) [inline examples/basic-env-vars.yaml]: I would suggest to remove the pod block with chaosArgs, as it's not what your example is intended to show.

Comment (dmartin35) [reply examples/basic-env-vars.yaml]: from lines 53 to 57

File: examples/basic-env-vars.yaml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: chaostoolkit-env
+  namespace: chaostoolkit-run
+data:
+  FIRST: "Jane"
+  SECOND: "Doe"
```
Comment (dmartin35) [inline examples/basic-env-vars.yaml]: I would suggest using more meaningful variables, like firstname and lastname, wdyt ?

**Review by dmartin35 — CHANGES_REQUESTED** [review]
overall manifest looks good

File: examples/basic-env-vars.yaml (hunk 1 above)
Comment (gtfisher) [reply examples/basic-env-vars.yaml]: fixed

File: examples/basic-env-vars.yaml (hunk 2 above)
Comment (gtfisher) [reply examples/basic-env-vars.yaml]: fixed

**Comment by dmartin35** [comment]
lgtm

**Pull request merged**